variables.tf

```diff
@@ -623,6 +623,15 @@ variable "microsoft_defender_enabled" {
   nullable    = false
 }
 
+variable "monitor_metrics" {
+  type = object({
+    annotations_allowed = optional(string)
+    labels_allowed      = optional(string)
+  })
+  description = "(Optional) Specifies a Prometheus add-on profile for the Kubernetes Cluster"
```

Comment:
Could we enrich the description like this:?

```hcl
description = <<-EOT
(Optional) Specifies a Prometheus add-on profile for the Kubernetes Cluster
object({
    annotations_allowed = "(Optional) Specifies a comma-separated list of Kubernetes annotation keys that will be used in the resource's labels metric."
    labels_allowed      = "(Optional) Specifies a comma-separated list of Kubernetes annotation keys that will be used in the resource's labels metric."
})
```

Reply:
done !